src/Serializer/AbstractItemNormalizer.php

```diff
@@ -298,6 +300,17 @@ private function denormalizeRelation(string $attributeName, PropertyMetadata $pr
         }
 
         if (!is_array($value)) {
+            // repeat the code so that IRIs keep working with the json format
```

Comment:
I'd like to test that `Accept` is `application/json` here, not sure if it's possible or wanted...

Reply:
you can just check the `$format` var.

Reply:
not really, `json` is the format for `hal`, `jsonld` and `json` no?

Reply:
No, it's `jsonld` for JSON-LD and `jsonhal` for HAL.

Reply:
Okay, my bad dunno why I thought so...